### ISSUE TITLE
Remove Node core warning

### ISSUE THREAD

**Issue by mcollina** [fastify/fastify-url-data#36]
## To Reproduce

Running `npm test` shows:

```js
(node:61289) [DEP0123] DeprecationWarning: Setting the TLS ServerName to an IP address is not permitted by RFC 6066. This will be ignored in a future version.
```

## Expected behavior

no warning

**Comment by richardlau** [comment]
```
-bash-4.2$ node --trace-warnings test/tests.test.js
TAP version 13
# Subtest: parses a full URI
    1..8
    ok 1 - should be equal
    ok 2 - should be equal
    ok 3 - should be equal
    ok 4 - should be equal
    ok 5 - should be equal
    ok 6 - should be equal
    ok 7 - should be equal
    ok 8 - should be equal
ok 1 - parses a full URI # time=62.503ms

# Subtest: parses a full URI in HTTP2
    1..9
    ok 1 - Key/cert successfully loaded
(node:47708) [DEP0123] DeprecationWarning: Setting the TLS ServerName to an IP address is not permitted by RFC 6066. This will be ignored in a future version.
    at Object.connect (_tls_wrap.js:1618:15)
    at Object.connect (internal/http2/core.js:3033:22)
    at request (/home/users/riclau/sandbox/github/fastify-url-data/node_modules/h2url/index.js:15:24)
    at Object.concat (/home/users/riclau/sandbox/github/fastify-url-data/node_modules/h2url/index.js:40:21)
    at /home/users/riclau/sandbox/github/fastify-url-data/test/tests.test.js:96:11
    at Http2SecureServer.wrap (/home/users/riclau/sandbox/github/fastify-url-data/node_modules/fastify/lib/server.js:81:9)
    at Object.onceWrapper (events.js:421:28)
    at Http2SecureServer.emit (events.js:315:20)
    at emitListeningNT (net.js:1347:10)
    at processTicksAndRejections (internal/process/task_queues.js:83:21)
    ok 2 - should be equal
    ok 3 - should be equal
    ok 4 - should be equal
    ok 5 - should be equal
    ok 6 - should be equal
    ok 7 - should be equal
    ok 8 - should be equal
    ok 9 - should be equal
ok 2 - parses a full URI in HTTP2 # time=49.06ms

1..2
# time=129.943ms
-bash-4.2$
```

**Comment by mcollina** [comment]
We need to change what we are doing in the http/2 test so that the warning is not emitted anymore.

**Comment by richardlau** [comment]
Changed the test to use a hostname (`localhost`) instead of an IP address (`127.0.0.1`):https://github.com/fastify/fastify-url-data/pull/37

**Comment by Eomm** [comment]
Fixed in #37 
Thks!
